testing_wasm/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "testing",
+  "version": "1.0.0",
+  "description": "Testing",
+  "scripts": {
+    "start": "node -r dotenv/config index.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Utkarsh Varma",
+  "license": "ISC",
```

Comment:
NO LICENSE

Reply:
fixed.

Reply:
Sorry to be clear should be "license": "NO LICENSE"

Reply:
Fixed.